pkg/crc/cluster/monitoring.go

```diff
@@ -20,20 +20,28 @@ func StartMonitoring(ocConfig oc.Config) error {
 		return err
 	}
 
+	indexForClusterMonitoringDeploymentKind := getIndexInOverridesForObjectName(cv, "cluster-monitoring-operator")
+	indexForClusterMonitoringCVOKind := getIndexInOverridesForObjectName(cv, "monitoring")
+
+	if indexForClusterMonitoringDeploymentKind != -1 && indexForClusterMonitoringCVOKind != -1 {
+		_, _, err = ocConfig.RunOcCommand("patch", "clusterversion/version",
+			"--type", "json",
+			"--patch", fmt.Sprintf(`'[{"op":"remove", "path":"/spec/overrides/%d"},{"op":"remove", "path":"/spec/overrides/%d"}]'`,
+				indexForClusterMonitoringDeploymentKind, indexForClusterMonitoringCVOKind-1))
+	}
+	return err
```

Comment:
Maybe ?

```
if indexForClusterMonitoringDeploymentKind != -1 && indexForClusterMonitoringCVOKind != -1 { 
  _, _, err = ocConfig.RunOcCommand()
  return err
}
return errors.New("Unable to find monitoring objects in CVO overrides")
```

Reply:
@guillaumerose if we want it blocking in case we are not able to find the kind index then sure but in this PR I didn't change the existing behavior of this function which doesn't take any action in case of not finding the object index.

Reply:
Oh I see, we need that in case we start-stop-start again the cluster. 👍